app/helpers/validators_helper.rb

```diff
@@ -75,7 +75,7 @@ def skipped_vote_percent(validator, batch)
   end
 
   def sort_software_versions(versions)
-    versions&.sort_by { |ver| Gem::Version.new(ver.keys.first)}&.reverse
+    versions&.sort_by { |ver| ver.values[0]["stake_percent"] }&.reverse
```

Comment:
There are two negative scenarios: 
1) versions are invalid and AppSignal will send an alert (we would notice that faster and think about some fix)
In this case I would suggest to make a change 
```suggestion
    versions.sort_by { |ver| ver.values[0]["stake_percent"] }.reverse
```

2) versions are invalid (e.g. nil) and we do not Rails to raise any problem (software versions list will be blank on homepage)
```suggestion
    Array(versions).sort_by { |ver| ver.values[0]["stake_percent"] }&.reverse
```

Reply:
IMO we should stick with the 1st scenario.

Reply:
I do not think this is a good idea. Raising an error will crash the front stats daemon and we don't want that. If something goes wrong we will still be able to see it via blank software versions component

Reply:
Yeah, I haven't notice that this is running in front stats daemon.